packages/block-editor/src/components/block-popover/index.js

```diff
@@ -88,6 +93,20 @@ function BlockPopover(
 
 		return {
 			getBoundingClientRect() {
+				if ( isZoomOut() ) {
+					const selectedRect =
+						getVisibleElementBounds( selectedElement );
+					const parentRect = getVisibleElementBounds(
+						selectedElement.parentElement
+					);
+					return new window.DOMRectReadOnly(
+						parentRect.left,
+						selectedRect.top,
+						parentRect.right - parentRect.left,
+						selectedRect.bottom - selectedRect.top
```

Comment:
```suggestion
						selectedRectHeight
```

Consider a variable to represent this and also to avoid having to look up the API of `DOMRectReadOnly`